samples/wifi/provisioning/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(wifi_ble_provisioning)
+
+set(WIFI_NRF_DRV_BASE ${WEST_TOPDIR}/nrf/drivers/wifi/nrf700x)
```

Comment:
I have an impression that this could be done in a better way. Could you please consult @tejlmand

Reply:
Please do not rely on path form the `WEST_TOPDIR`, cause that puts to much restriction on location of the Zephyr module.

sdk-nrf can be referenced as: `${ZEPHYR_NRF_MODULE_DIR}`.

That said, why do you set the CMake variable `WIFI_NRF_DRV_BASE` ?
I don't see this variable used anywhere.

Reply:
Will track this issue in conversation below.